src/main/resources/templates/section.html

```diff
@@ -26,9 +26,9 @@
         <div class="container">
             <table class="table">
                 <tr >
-                    <td th:text="${section.course.department + ' ' + section.course.number}"></td>
-                    <td th:text="${section.course.name}"></td>
-                    <td><a class="badge badge-info" th:href="@{/course/{id}(id=${section.course.id})}">view</a></td>
+                    <td th:text="${course.department} + ' ' + ${course.number}"></td>
```

Comment:
would you explain your thoughts on changing these from section.course to simply course?

Reply:
Sometime the section is null. This makes null pointer exception

Reply:
ok sounds good to me, we could also simply check for this inside of the controller as well but I am fine with this if it works.